### DISCUSSION TITLE
jumper t lite

### DISCUSSION THREAD

**Discussion by grzegorz2012** [EdgeTX/edgetx#4824] [Q&A]
Hello, I have an extended t lite v2 jumper with a new CKS32F processor, I have added 2 switches and 2 potentiometers.
I use jumper 12 pro as an input and everything works fine until version 2.85.
I cannot update to higher versions of edge tx, bad processor message. Everything worked fine on the v1 ARM board. Would it be possible to make the new processor fully compatible with current edgetx versions?

**Comment by pfeerick** [comment]
> extended t lite v2 jumper with a new CKS32F processor

Where can we find more details about this?

If you're getting the "bad MCU" message... you could try the tlitef4 build instead and see if that works, but they were never supposed to have eventuated.
